docs/source/models.md

```diff
@@ -1,53 +1,77 @@
 # The Transaction, Block and Vote Models
 
-Transactions, blocks and votes are represented using JSON documents with the following models (schemas). See [the section on cryptography](cryptography.html) for more information about how we calculate hashes and signatures.
+BigchainDB stores all its records in JSON documents.
+
+The three main kinds of records are transactions, blocks and votes. 
+_Transactions_ are used to register, issue, create or transfer things (e.g. assets). Multiple transactions are combined with some other metadata to form _blocks_. Nodes append _votes_ to blocks. This section is a reference on the details of transactions, blocks and votes.
```

Comment:
This was added after a discussion with Bruce...  A transaction with no inputs can have many uses, and they can't all be captured with one verb, or at least I don't think they can.

In any case, if we remove the "operation" from the transaction data model (as suggested in issue #340), we're free to use whatever verb we like when describing what one of these input-free transactions is doing.